server/db/middleware/auth.js

```diff
@@ -0,0 +1,38 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import { GET_USER_BY_ID } from '../helpers/query';
+import ST from '../../helpers/status';
+import MSG from '../../helpers/res_messages';
+class Auth {
+    constructor(){
+        dotenv.config();
+    }
+    async verifyToken(req, res, next){
+        let token =  req.headers['epic-mail-access-token'];
+        if(!token){
+            return res.status(ST.BAD_REQUEST).send({
+                "status" : ST.BAD_REQUEST,
+                "error" : "Token is not provided"
+            });
+        }
+        const decoded = await jwt.verify(token, process.env.JWT_SECRET);
+        const { rows } = await db.query(GET_USER_BY_ID, [decoded.userid]);
+        if(!rows[0]) {
+          return res.status(ST.BAD_REQUEST).send({ 
+              "status":ST.BAD_REQUEST,
```

Comment:
Have you set up eslint for your project?

Reply:
Yeah, but I did not enforce syntax integrity